static/src/server.ts

```diff
@@ -172,15 +200,41 @@ function getTextLength(text: string): number {
   };
 };
 
-// Gets the svg element for a scatter tile.
-function getScatterTileSvg(
+// Gets a list of source objects with name and url from a set of source urls.
+function getSources(sources: Set<string>): { name: string; url: string }[] {
+  return Array.from(sources).map((src) => {
+    return {
+      name: urlToDomain(src),
+      url: src,
+    };
+  });
+}
+
+// Processes and serializes a svg for a chart.
+function getProcessedSvg(chartSvg: SVGSVGElement): string {
+  if (!chartSvg) {
+    return "";
+  }
+  // Set the font for all the text in the svg
+  chartSvg.querySelectorAll("text").forEach((node) => {
+    node.setAttribute("font-family", FONT_FAMILY);
+    node.setAttribute("font-size", FONT_SIZE);
+  });
```

Comment:
comment on why the font size was not size in the original svg?

Reply:
done